client/signup/steps/reader-landing/style.scss

```diff
@@ -1,8 +1,111 @@
+.reader-landing {
+	margin: 0 20px;
+}
+
 .reader-landing__step-content {
-	text-align: center;
+	text-align: left;
+	padding-bottom: 20px;
+}
+
+.reader-landing .formatted-header {
+	margin-bottom: 16px;
+
+	@include breakpoint( ">660px" ) {
+		display: flex;
+		justify-content: center;
+		max-width: 800px;
+		flex-direction: column;
+		margin-left: auto;
+		margin-right: auto;
+	}
+}
+
+.reader-landing .formatted-header__title {
+	font-size: 1.8em;
+	text-align: left;
+	padding: 0;
+
+	@include breakpoint( ">660px" ) {
+		font-size: 2.4em;
+		text-align: center;
+		margin-top: 0;
+	}
+}
+
+.reader-landing .formatted-header__subtitle {
+	text-align: left;
+	padding: 0;
+
+	@include breakpoint( ">660px" ) {
+		text-align: center;
+		margin-top: 10px;
+		margin-bottom: 0;
+		font-size: 1.1em;
+	}
+}
+
+.reader-landing__button-wrapper {
+	@include breakpoint( ">660px" ) {
+		text-align: center;
+	}
+}
+
+.reader-landing__button {
+	width: 100%;
+
+	@include breakpoint( ">660px" ) {
+		width: inherit;
+		text-align: center;
+	}
 }
 
-.reader-landing__step-content-illustration {
-	max-height: 140px;
-	margin-bottom: 20px;
-}
\ No newline at end of file
+.reader-landing__features {
+	margin-top: 30px;
+}
+
+.reader-landing__feature {
+	margin-bottom: 30px;
+
+	@include breakpoint( ">660px" ) {
+		display: flex;
+	}
+}
+
+@include breakpoint( ">660px" ) {
+	.reader-landing__feature-image, .reader-landing__feature-detail {
+		flex: 50%;
+	}
+
+	.reader-landing__feature-image {
+		align-self: center;
+		max-width: 50%;
+	}
+}
+
+.reader-landing__feature-detail {
+	margin-top: 10px;
+
+	@include breakpoint( ">660px" ) {
+		padding-left: 10px;
+	}
+}
+
+.reader-landing__feature-heading {
+	font-size: 1.2em;
+	font-weight: bold;
+
+	@include breakpoint( ">660px" ) {
+		font-size: 1.6em;
+	}
+}
+
+.reader-landing__feature-description {
+	color: darken( $gray, 20 );
+	font-size: 14px;
```

Comment:
This is the only one in `px`. I'm not sure what the standard is though, but it might be `px` instead of `em`.

Reply:
Good spot. I'd noticed that the standard signup headings were recently specified in `em`, so decided to stick with that when overriding them:

https://github.com/Automattic/wp-calypso/commit/83d8b8613bf4f3c2094c804cd310b07529808abe#diff-2e2cfa9cf99526a24547e9cddf63a0ab